README.md

```diff
@@ -110,11 +129,103 @@ $ ab build                                       \  # this is the command
      this-is-my-image                            \  # and finally, target image name
 ```
 
+At some point, I should figure out a file-format where these would live (ansible variables? a dedicated file?).
```

Comment:
Variables seem like the most "ansible" way to do this. Something like this would do the trick:
```yaml
build_env_vars:
    SOME: VALUE
container_image_labels:
    some: other-value
```

If those are already in variables, we could also put the base image and the tag into there, reducing the command to just `ab build <path to playbook>`

Reply:
Yup, absolutely. Let's continue the discussion in a separate issue: https://github.com/TomasTomecek/ansible-bender/issues/10